src/main/java/org/team1540/robot2019/commands/drivetrain/PercentDrive.java

```diff
@@ -0,0 +1,24 @@
+package org.team1540.robot2019.commands.drivetrain;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.team1540.robot2019.OI;
+import org.team1540.robot2019.Robot;
+
+public class PercentDrive extends Command {
```

Comment:
Maybe specify `PercentTankDrive` or something?

Also I think I actually wrote a ROOSTER input for tank drive at some point

Reply:
Done.